src/main/scala/mesosphere/marathon/api/akkahttp/PathMatchers.scala

```diff
@@ -49,6 +53,10 @@ object PathMatchers {
     override def apply(path: Path) = iter(Nil, path)
   }
 
+  case object PodsPathIdLike extends PathMatcher1[String] {
```

Comment:
Why do we need a matcher which doesn't match anything?

Reply:
I've updated it.